content/vault/v1.18.x/content/docs/upgrading/upgrade-to-1.16.x.mdx

```diff
@@ -18,6 +18,48 @@ Vault 1.15. **Please read carefully**.
 
 ## Important changes
 
+### JSON Payload Limits
```

Comment:
Can you make this a partial? It will have to go in the global partials, but it seems worth it if we ever have to update it.

I think it also needs to be in the v1.18.x/content/docs/upgrading upgrade to 1.16.x guide, moreseo than in this file.

@schavis does that make sense for this kind of content?

Reply:
Yes. Once we finish the cleanup work, everything will be a partial since even new behaviors can affect past versions (like this one).